singer/src/main/java/com/pinterest/singer/writer/s3/ObjectUploaderTask.java

```diff
@@ -0,0 +1,71 @@
+package com.pinterest.singer.writer.s3;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+import software.amazon.awssdk.services.s3.model.PutObjectResponse;
+
+import java.io.File;
+
+public class ObjectUploaderTask {
+    public static final int MAX_RETRIES = 5;
```

Comment:
where is this static final int being used if the maxRetries config is passed in by the constructor?

Reply:
```
if (s3WriterConfig.getMaxRetries() > 0) {
      this.maxRetries = s3WriterConfig.getMaxRetries();
    } else {
      this.maxRetries = ObjectUploaderTask.MAX_RETRIES;
    }
```
I use it in S3 Writer code to make sure.